src/explorer/CosmosDBTreeItem.ts

```diff
@@ -72,21 +72,41 @@ export class CosmosDBTreeItem extends AzureParentTreeItem<ISiteTreeRoot> {
         // tslint:disable-next-line:strict-boolean-expressions
         connectionsUnit.cosmosDB = connectionsUnit.cosmosDB || [];
         if (!connectionsUnit.cosmosDB.find((x: string) => x === connectionToAdd)) {
+            const connectionStringValue = (<string>await vscode.commands.executeCommand('cosmosDB.api.getConnectionString', connectionToAdd));
+
+            const allSettingsNames: string[] = [];
+            const appSettingsToCreate = new Map<string, string>();
+            if (/^mongo/i.test(connectionStringValue)) {
+                allSettingsNames.push('MONGO_URL');
+                appSettingsToCreate.set(allSettingsNames[0], connectionStringValue);
```

Comment:
I don't want you referencing `allSettingsNames[0]`. Just create a separate variable for mongoUrl and reference that explicitly. We don't know how allSettingsNames could change in the future and I don't want to rely on the ordering of the array. For example, maybe someone adds a default setting to both mongo AND docdb which means `allSettingsNames[0]` would resolve to that default setting instead of 'MONGO_URL'


Mostly I'm worried about a scenario in the future where someone adds a default value to `allSettingsNames` and then